src/indexing_sources/source_pgsql.cpp

```diff
@@ -111,6 +115,8 @@ struct CSphSource_PgSQL : CSphSource_SQL
 	DWORD			SqlColumnLength ( int iIndex ) final;
 	const char *	SqlColumn ( int iIndex ) final;
 	const char *	SqlFieldName ( int iIndex ) final;
+	const char *	SqlColumnStream ( int iIndex, DWORD &iStreamLength ) final;
```

Comment:
minor: according to our coding rules, unsigned should be prefixed with 'u'. I.e. `uStreamLength` here, as it is DWORD